.github/workflows/publish.yml

```diff
@@ -14,6 +14,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - name: Fail if we are attempting to run on the master branch
+      if: ${{GITHUB.REF_NAME == 'master'}}
```

Comment:
This is Wizden-specific, and therefore this check should also check what repo this is being run off.

![image](https://github.com/user-attachments/assets/4333a894-555e-46ba-93fa-c0463e243fe2)